src/main/java/org/openstreetmap/atlas/generator/tools/filesystem/FileSystemHelper.java

```diff
@@ -428,7 +447,7 @@ public static WritableResource writableResource(final String path,
         }
         catch (final Exception e)
         {
-            throw new CoreException("Unable to read {}", hadoopPath, e);
+            throw new CoreException(UNABLE_TO_READ, hadoopPath, e);
```

Comment:
💯thanks for cleaning up the Strings here!

Reply:
Eclipse Sonar Lint was complaining about it, and I know that `atlas-checks` uses sonar, so I figured I'd save a round trip on that. Hopefully.